internal/manifests/manifestutils/constants.go

```diff
@@ -48,10 +48,15 @@ const (
 	// PortJaegerUI declares the port number of the Jaeger UI HTTP port.
 	PortJaegerUI = 16686
 
-	// OAuthProxyPortName declares the name of the Jaeger UI oauth proxy HTTP port.
-	OAuthProxyPortName = "oauth-proxy"
-	// OAuthProxyPort declares the port number of the Jaeger UI oauth proxy HTTP port.
-	OAuthProxyPort = 8443
+	// OAuthJaegerUIProxyPortHTTP declares the port number of the Jaeger UI oauth proxy HTTPS port.
```

Comment:
The HTTPS is the only one that we use, but oauth proxy open one for http and other for https. I can try to find a way to disable HTTP. If I don't specify a port for HTTP it will try to open the same port for both oauth proxies.